arrows/vxl/hashed_image_classifier_filter.cxx

```diff
@@ -0,0 +1,168 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "hashed_image_classifier.h"
+#include "hashed_image_classifier_filter.h"
+
+#include <arrows/vxl/image_container.h>
+
+#include <vital/vital_config.h>
+
+#include <vil/algo/vil_threshold.h>
+#include <vil/vil_convert.h>
+#include <vil/vil_image_view.h>
+#include <vil/vil_math.h>
+#include <vil/vil_plane.h>
+
+#include <cstdlib>
+#include <limits>
+#include <type_traits>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+// ----------------------------------------------------------------------------
+/// Private implementation class
+class hashed_image_classifier_filter::priv
+{
+public:
+  // --------------------------------------------------------------------------
+  priv( hashed_image_classifier_filter* parent ) : p{ parent }
+  {
+  }
+
+  // Convert the type
+  template < typename ipix_t > vil_image_view< ipix_t >
+  convert( vil_image_view_base_sptr& view );
+
+  // Scale and convert the image
+  bool
+  load_model();
+
+  hashed_image_classifier_filter* p;
```

Comment:
```suggestion
  hashed_image_classifier_filter* const p;
```
This will prevent default copy/assignment, but that's a *feature* since those would do the wrong thing anyway.